### ISSUE TITLE
Difference batch size with different results

### ISSUE THREAD

**Issue by Yaoufly** [xiaozhen228/VCP-CLIP#4]
hi, @xiaozhen228 

Thanks for your work.
But the re-produced performance with different batch size is very different. I trained on the VisA, if I used default batch_size 32, I could get metrics similar to your paper, but if I used 16, the metrics are much lower.
Could you please explain the reason

**Comment by xiaozhen228** [comment]
> hi, @xiaozhen228
> 
> Thanks for your work. But the re-produced performance with different batch size is very different. I trained on the VisA, if I used default batch_size 32, I could get metrics similar to your paper, but if I used 16, the metrics are much lower. Could you please explain the reason
Hi, a smaller batch size implies that the model will update its parameters more frequently within one epoch under the same learning rate. Generally, to achieve the same training effect, the learning rate needs to be reduced. Moreover, supervised prompt learning may lead to the model overfitting the auxiliary training set, thereby reducing its generalization ability in zero-shot cross-product segmentation, which is a potential reason for this issue. We recommend using the same hyperparameters as provided in our experiments.

**Comment by Qi-Ran** [comment]
> hi, @xiaozhen228
> 
> Thanks for your work. But the re-produced performance with different batch size is very different. I trained on the VisA, if I used default batch_size 32, I could get metrics similar to your paper, but if I used 16, the metrics are much lower. Could you please explain the reason
How did you manage to reproduce results consistent with the paper? The results I reproduced differ significantly.

**Comment by goodzxt** [comment]
> hi, @xiaozhen228
> 
> Thanks for your work. But the re-produced performance with different batch size is very different. I trained on the VisA, if I used default batch_size 32, I could get metrics similar to your paper, but if I used 16, the metrics are much lower. Could you please explain the reason

I set the batchsize to 8 and adjusted the learning rate from 8e-5 to 1e-6, neither of which achieved performance metrics close to those achieved with the original setting of batchsize of 32. May I ask if you finally solved the problem?
